dev/scripts/platform_upload.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+curl -X POST http://localhost:9001/control/upload
```

Comment:
I got error log on server side after calling it. Response is 200. It's because of code in listener.go:
~~~go
vmaasData, resp, err := vmaasClient.UpdatesApi.AppUpdatesHandlerV2PostPost(ctx, &vars)
utils.Log("data", vmaasData, "res", resp).Error("VMAAS query complete")
~~~
I see no reason to log it as Error. It should be tested to err var to be nil.

Reply:
Fixed